tests/integration/ceph_base_object_test.go

```diff
@@ -160,7 +160,7 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	s3endpoint, _ := helper.ObjectClient.GetEndPointUrl(namespace, storeName)
 	s3AccessKey, _ := helper.BucketClient.GetAccessKey(obcName)
 	s3SecretKey, _ := helper.BucketClient.GetSecretKey(obcName)
-	s3client, err := rgw.NewS3Agent(s3AccessKey, s3SecretKey, s3endpoint, true)
+	s3client, err := rgw.NewS3Agent(s3AccessKey, s3SecretKey, s3endpoint, true, false)
```

Comment:
Do we have any integration test that verifies functionality with TLS enabled? This is the more complex config; if we only test it one way, it seems to me that it might be better to test with TLS to verify we aren't missing anything critical.

Reply:
@BlaineEXE : Currently creation fails for TLS environment, I have fix for that but waiting for this PR to merge. I will add the test case in that scenario.